frontend/cypress/tests/chartView/chartView_zoomFeature.cy.js

```diff
@@ -39,3 +39,260 @@ describe('zoom features in code view', () => {
       .should('be.visible');
   });
 });
+
+describe('date changes in chart view should reflect in zoom', () => {
+  it('changing \'since\' date range changes the zoom view', () => {
+    cy.get('div.mui-textfield.search_box > input:visible')
+      .should('be.visible')
+      .type('eugene');
+
+    cy.get('input[name="since"]:visible')
+      .type('2018-06-11');
+
+    cy.get('.icon-button.fa-list-ul')
+      .should('be.visible')
+      .first()
+      .click();
+
+    cy.get('#tab-zoom')
+      .should('be.visible');
+
+    cy.get('#tab-zoom .ramp .ramp__slice')
+      .last()
+      .invoke('attr', 'title')
+      .should('eq', '[2018-06-12] Setup AppVeyor CI (#142): +19 -0 lines ');
```

Comment:
For the date tests, the functionality/requirement we're interested in testing is that changing the date in the summary view will filter the commits in zoom to only be between the start/end dates. So instead of asserting that the last commit is this specific commit, would it be better to do something like loop through the commits and asserting that their date falls between the since/until dates? 

This would also make this test not rely on the correctness of the sort.